ts/screens/wallet/payment/PickPspScreen.tsx

```diff
@@ -102,66 +104,98 @@ class PickPspScreen extends React.Component<Props> {
     this.props.loadAllPsp(idWallet, idPayment);
   }
 
+  private headerItem = (
+    <View style={styles.padded}>
+      <View style={styles.line1}>
+        <LabelSmall weight="Regular" color="bluegrey">
+          {I18n.t("wallet.pickPsp.provider")}
+        </LabelSmall>
+        <LabelSmall weight="Regular" color="bluegrey">
+          {`${I18n.t("wallet.pickPsp.maxFee")} (€)`}
+        </LabelSmall>
+      </View>
+      <View spacer />
+      <ItemSeparatorComponent noPadded />
+    </View>
+  );
+
   private getListItem = (psp: ListRenderItemInfo<Psp>) => {
     const { item } = psp;
+
     return (
       <TouchableDefaultOpacity
         onPress={() => this.props.pickPsp(item.id, this.props.allPsps)}
         style={styles.itemContainer}
       >
         <View style={styles.line1}>
-          <Image
-            style={styles.flexStart}
-            resizeMode={"contain"}
-            source={{ uri: item.logoPSP }}
-          />
-          <IconFont
-            name={"io-right"}
-            size={ICON_SIZE}
-            color={customVariables.contentPrimaryBackground}
-          />
+          {!this.state.hasImageLoadingError ? (
+            <Image
+              style={styles.imageProvider}
+              resizeMode="contain"
+              source={{ uri: item.logoPSP }}
+              onError={this.onErrorImageLoading}
+            />
+          ) : (
+            <Body>{item.businessName}</Body>
+          )}
+          <View style={styles.feeContainer}>
+            <Label>{formatNumberCentsToAmount(item.fixedCost.amount)}</Label>
+            <IconFont
+              name="io-right"
+              size={ICON_SIZE}
+              color={customVariables.contentPrimaryBackground}
+            />
+          </View>
         </View>
-        <Text style={styles.feeText}>
-          {`${I18n.t("wallet.pickPsp.maxFee")} `}
-          <Text bold={true} style={styles.feeText}>
-            {formatNumberCentsToAmount(item.fixedCost.amount)}
-          </Text>
-        </Text>
       </TouchableDefaultOpacity>
     );
   };
 
   public render(): React.ReactNode {
     const availablePsps = orderPspByAmount(this.props.allPsps);
 
+    const backButtonProps = {
+      block: true,
+      primary: true,
+      bordered: true,
+      onPress: this.props.navigateBack,
+      title: I18n.t("global.buttons.back")
+    };
+
     return (
       <BaseScreenComponent
         goBack={true}
-        headerTitle={I18n.t("wallet.pickPsp.title")}
+        headerTitle={I18n.t("wallet.pickPsp.headerTitle")}
         contextualHelpMarkdown={contextualHelpMarkdown}
         faqCategories={["payment"]}
       >
-        <Content noPadded={true}>
-          <View spacer={true} />
+        <Content noPadded>
```

Comment:
What do you think about use a `SafeAreaView` component instead? 
```suggestion
        <SafeAreaView style={IOStyles.flex}>
```